lms/static/scripts/frontend_apps/components/GroupConfigSelector.js

```diff
@@ -106,93 +193,79 @@ export default function GroupConfigSelector({
       useGroupSet: false,
       groupSet: null,
     });
-  };
+  }, [onChangeGroupConfig]);
+
+  const onGroupSelectChange = useCallback(
+    /**
+     * @param {string|null} groupSetID
```

Comment:
If there is only one param in a callback I usually write it all on one line for compactness (`/** @param {string|null} groupSetID */`).